### ISSUE TITLE
Android: Billing is not initialized

### ISSUE THREAD

**Issue by mklilley** [AlexDisler/cordova-plugin-inapppurchase#3]
Hello again.  I am having some trouble with android (Google nexus 4 running android 5.1.1).  When i turn on my app (making it with ionic 1.2.4 and cordova 6.0.0) I wait for the device to load and then I do a restore purchase.  I get the error `Object {code: -3, message: "Billing is not initialized"}`. 

I dug around a bit and ended up inside inAppBillingV3.java.  I was unsure where the problem was so I plugged my phone into the computer and ran logcat so i could see all the logs.  What I see is this:

```
D/google.payments(14273): manifest:{ "play_store_key": "my-specific-key" }                                                                                                                                           
D/google.payments(14273): executing on android                                                                                                                                                    
D/google.payments(14273): Error: Billing is not initialized   

```

I put some logs into the initialization function but nothing comes out on the console which makes me think maybe the plugin isnt properly initialized.

Apologies if im missing something really simple here.  Really appreciate your help.

Thanks,

Matt

**Comment by AlexDisler** [comment]
Hi,
Try calling ```getProducts()``` before making or restoring purchases.

**Comment by mklilley** [comment]
Hi.  I just saw that....should have looked there first, oops, long day.  May I ask, is there a reason you chose not to have init as a separate function to call?  I am just thinking, should one need to get the product information in order to restore the products? What do you think?

**Comment by AlexDisler** [comment]
I've updated the readme to be more clear about it, thanks for letting me know.

I think ```restorePurchases()``` should call init internally and that would resolve this issue. (init can be called multiple times without consequences). This way we can avoid adding another function that will be required only for Android. I prefer to keep the api consistent for both platforms.

**Comment by mklilley** [comment]
No problem, thanks for a) making the plugin and b) for getting back to me so fast.  I am not such an expert on these things so people like you really help me a lot. :-)

Yes I think what you suggest about putting init inside of restore make a lot of sense.

**Comment by AlexDisler** [comment]
I've implemented it in the new version (0.2.0), you can try calling ```restorePurchases()``` without ```getProducts()``` first. Please reopen if you have any issues.

**Comment by dhavalkakkad** [comment]
Hello guys, 

I am using cordova 6.3.1
Ionic 1.3 
Node 4.2.4
Play store key is set in www/manifest.json

I have made 2 products in test mode to my google playstore account. Products are active, I am just trying to load products, not buying or restoring. My device is Android 6.0. 

Getting the same error : Billing cannot be initialized. 

Is there anything I am missing ?
